lib/Runtime/Language/JavascriptConversion.cpp

```diff
@@ -1463,3 +1463,17 @@ using namespace Js;
         return NumberUtilities::TryToInt64(length);
     }
 
+    JavascriptBigInt *JavascriptConversion::ToBigInt(Var aValue, ScriptContext* scriptContext)
+    {
+        Assert(scriptContext->GetThreadContext()->IsScriptActive());
+        switch (JavascriptOperators::GetTypeId(aValue))
+        {
+        case TypeIds_BigInt:
+            break;
+        default:
+            AssertMsg(JavascriptOperators::IsObject(aValue), "bad type object in conversion ToBigInt");
+            AssertMsg(FALSE, "wrong call in ToBigInt, no dynamic objects should get here");
+            JavascriptError::ThrowError(scriptContext, VBSERR_OLENoPropOrMethod);
```

Comment:
Not sure that this error code should be used here -- perhaps simply `AssertOrFailFast(typeId == TypeIds_BigInt)` can replace this entire switch?